examples/external/java/read-keys-example/gradlew

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env sh
```

Comment:
Do we need this really several times? Is there no way to use ./src/bindings/jna/gradlew?

Reply:
The gradle wrapper has to reside in the root gradle project folder. This file is managed by gradle. We could do some symlinking, but i do not see any benefit.